src/main/java/org/jenkinsci/plugins/mesos/MesosSingleUseSlave.java

```diff
@@ -57,6 +57,22 @@ public MesosSingleUseSlave() {
     @Override
     public void setUp(Context context, Run<?, ?> build, FilePath workspace, Launcher launcher, TaskListener listener, EnvVars initialEnvironment) throws IOException, InterruptedException {
         context.setDisposer(new MesosSingleUseSlaveDisposer());
+
+        Computer computer = workspace.toComputer();
+        if (computer != null) {
+            if (MesosComputer.class.isInstance(computer)) {
+                String msg = "Marking " + computer.getName() + " as single-use.";
+                LOGGER.warning(msg);
```

Comment:
not sure why this is a `LOGGER.warning`. this could just be `Logger.info` since this is intentional?